jbpm-console-ng-human-tasks/jbpm-console-ng-human-tasks-client/pom.xml

```diff
@@ -93,6 +93,19 @@
       <artifactId>jbpm-console-ng-generic-client</artifactId>
       <type>jar</type>
     </dependency>
+
+    <!-- Test -->
+    <dependency>
+      <groupId>org.mockito </groupId>
+      <artifactId>mockito-all</artifactId>
```

Comment:
Please avoid the usage of mockito-all (uberjar) - instead use mockito-core. 
See https://issues.jboss.org/browse/DROOLS-999 for details.